CHANGELOG.md

```diff
@@ -2,6 +2,37 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [2.20.0](https://github.com/serverless/serverless/compare/v2.19.0...v2.20.0) (2021-01-21)
+
+### Features
+
+- **AWS Lambda:**
+  - Add support for building Docker images ([#8725](https://github.com/serverless/serverless/issues/8725)) ([789c2e3](https://github.com/serverless/serverless/commit/789c2e35ab26b7e8dc0679f36110234fb899d57c)) ([Piotr Grzesik](https://github.com/pgrzesik))
+  - Add support for image config ([#8778](https://github.com/serverless/serverless/issues/8778)) ([9a55537](https://github.com/serverless/serverless/commit/9a5553742a3c3ebee03bfab5663a9183d5c228ba)) ([Piotr Grzesik](https://github.com/pgrzesik))
+- **CLI:** Support absolute path at "--config" ([#8793](https://github.com/serverless/serverless/pull/8793)) ([aea9f93](https://github.com/serverless/serverless/commit/aea9f9397c3806edda9958e324678af168b46b2f)) ([Mariusz Nowak](https://github.com/medikoo))
```

Comment:
Should that be a feature or a fix? @medikoo

Reply:
Good question. I believe this should be skipped. As it's an improvement for refactor introduced with #8770). So it was an improvement in context of `master`, but it's not any improvement against v2.19.0 (and this CHANGELOG should be based against that)